services/repository.py

```diff
@@ -87,9 +131,11 @@ def get_repo_provider_service(
             secret=get_config(service, "client_secret"),
         ),
         on_token_refresh=get_token_refresh_callback(token_owner),
-        fallback_installations=installation_info.get("fallback_installations")
-        if installation_info
-        else None,
+        fallback_installations=(
+            installation_info.get("fallback_installations")
+            if installation_info
+            else None
```

Comment:
So is to have a fallback option, the gh app we think doesn't have that repo selected correct? 

If there is a fallback, and it indeed doesn't have the repo in that fallback, would it eventually error out?

Reply:
This was a lint change. It's not directly related to the change here.

We only have fallback installations if the user in question has multiple configured apps (dedicated cloud). In this case you would have installations that cover the repo (more than 1 actually) and not raise RepoBotNotConfigured.

Reply:
Oh I see that makes sense, okok ty